### PR TITLE
base: add creation of weston home directory

### PR DESCRIPTION
**Pull request by Tim-Anderson**
Signed-off-by: Tim Anderson <tim.anderson@foundries.io>

### CODE REVIEW

File: meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend [hunk 1]
```diff
@@ -25,9 +27,12 @@ uncomment() {
 do_install_append_lmp-wayland() {
     install -d ${D}${datadir}/weston/backgrounds
     install -d ${D}${datadir}/weston/icon
+    install -d ${D}/${nonarch_libdir}/tmpfiles.d
 
     install -m 0644 ${WORKDIR}/utilities-terminal.png ${D}${datadir}/weston/icon/utilities-terminal.png
     install -m 0644 ${WORKDIR}/background.jpg ${D}${datadir}/weston/backgrounds/background.jpg
+    install -m 0644 ${WORKDIR}/tmpfiles.conf ${D}/${nonarch_libdir}/tmpfiles.d/weston.conf
+
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend:35]: No need for this extra line.

File: meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend [hunk 2]
```diff
@@ -25,9 +27,12 @@ uncomment() {
 do_install_append_lmp-wayland() {
     install -d ${D}${datadir}/weston/backgrounds
     install -d ${D}${datadir}/weston/icon
+    install -d ${D}/${nonarch_libdir}/tmpfiles.d
 
     install -m 0644 ${WORKDIR}/utilities-terminal.png ${D}${datadir}/weston/icon/utilities-terminal.png
     install -m 0644 ${WORKDIR}/background.jpg ${D}${datadir}/weston/backgrounds/background.jpg
+    install -m 0644 ${WORKDIR}/tmpfiles.conf ${D}/${nonarch_libdir}/tmpfiles.d/weston.conf
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend:34]: No need for '/' before ${nonarch_libdir}.

**Review by angolini — APPROVED** [review]

File: meta-lmp-base/recipes-graphics/wayland/weston-init/lmp-wayland/tmpfiles.conf [hunk 3]
```diff
@@ -0,0 +1 @@
+d /home/weston 0755 63 63 -
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-graphics/wayland/weston-init/lmp-wayland/tmpfiles.conf:1]: Use weston instead of 63 here.

**Comment by ricardosalveti** [comment]
Also please add at least one line as part of your commit message body, explaining why.

File: meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend [hunk 4]
```diff
@@ -25,9 +27,11 @@ uncomment() {
 do_install_append_lmp-wayland() {
     install -d ${D}${datadir}/weston/backgrounds
     install -d ${D}${datadir}/weston/icon
+    install -d ${D}/${nonarch_libdir}/tmpfiles.d
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend:30]: No need for '/' here.

**Review by ricardosalveti — COMMENTED** [review]
LGTM, one minor nit.

**Review by ricardosalveti — APPROVED** [review]

File: meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend (hunk 4 above)
Comment (Tim-Anderson) [reply meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend]: done

**Pull request merged**